lib/public/components/common/selection/SelectionModel.js

```diff
@@ -154,27 +169,32 @@ export class SelectionModel extends Observable {
      *
      * Depending on the selector, this may be a filtered subset of all the available options
      *
-     * @return {RemoteData|SelectionOption[]} the list of options
+     * @return {RemoteData<SelectionOption[], *>|SelectionOption[]} the list of options
      */
     get options() {
-        return this._availableOptions;
-    }
-
-    /**
-     * Return the list of all the available options
-     *
-     * @return {RemoteData|SelectionOption[]} the list of available options
-     */
-    get availableOptions() {
-        return this._availableOptions;
+        /**
+         * Add the default options to the list of given options
+         *
+         * @param {SelectionOption[]} options the options to which default selection should be added
+         * @return {SelectionOption[]} the options list including default options
+         */
+        const addDefaultToOptions = (options) => [
+            ...options,
+            ...this.optionsSelectedByDefault.filter((defaultOption) => !options.find(({ value }) => defaultOption.value === value)),
+        ];
+
+        return this._availableOptions instanceof RemoteData
+            ? this._availableOptions.apply({ Success: addDefaultToOptions })
+            : addDefaultToOptions(this._availableOptions);
     }
 
     /**
      * Defines the list of available options
      *
-     * @param {RemoteData|SelectionOption[]} availableOptions the new available options
+     * @param {RemoteData<SelectionOption[], *>|SelectionOption[]} availableOptions the new available options
+     * @return {void}
```

Comment:
It's not quite true, as actual `availableOptions` are those plus defaults

Reply:
Available options are the options that are actually available, not the options that are displayed to the user. Again, my english is probably lacking here